worker_node/job/send.js

```diff
@@ -164,11 +166,18 @@ class Send extends Base {
         continue;
       }
 
+      const tx = JSON.parse(JSON.stringify(baseTx));
       sendTxPromiseList.push(
           new Promise((resolve, reject) => {
             setTimeout((timestamp) => {
-              baseTx.timestamp = timestamp;
-              this.#ain.sendTransaction(baseTx).then(result => {
+              tx.timestamp = timestamp;
+              if (randomPath) {
+                tx.operation.ref = `${tx.operation.ref}/${i}`;
```

Comment:
It seems the paths are not really 'random' but rather 'consecutive'. Could you consider renaming it to consecutivePath OR using something like Math.random() or uuid?

Reply:
Looks good! I'll change it!